classes/image_types_ostree.bbclass

```diff
@@ -16,7 +16,8 @@ GARAGE_PUSH_RETRIES_SLEEP ??= "0"
 
 SYSTEMD_USED = "${@oe.utils.ifelse(d.getVar('VIRTUAL-RUNTIME_init_manager') == 'systemd', 'true', '')}"
 
-IMAGE_CMD_TAR = "tar --xattrs --xattrs-include=*"
+# tar needs selinux arg to preserve selinux labels, so check if distro is enabled with selinux 
+IMAGE_CMD_TAR ?= "tar ${@bb.utils.contains('DISTRO_FEATURES', 'selinux', '--selinux', '', d)} --xattrs --xattrs-include=*"
```

Comment:
Here I wonder if this is not somehow conflict with the other IMAGE_CMD_TAR set we have at https://github.com/openembedded/openembedded-core/blob/master/meta/classes-recipe/image_types.bbclass#L146, because this line is changing from = to ?=, and image_types seems to be parsed first.

We can probably just use a different variable name to avoid conflicts, like OSTREE_CMD_TAR or similar.

Reply:
good point! `?=` doesn't make much sense in this case, keep just `=` shoud be better.

Reply:
or we can just add the args we need in this class but if someone change the oe-core IMAGE_CMD_TAR it can fail, although it is unlikely to change the default `IMAGE_CMD_TAR ?= "tar"`.
```
IMAGE_CMD_TAR:append = "${@bb.utils.contains('DISTRO_FEATURES', 'selinux', ' --selinux', '', d)} --xattrs --xattrs-include=*"
```

Reply:
Ok based on Ricardo's comment it make sense to remove  "? "  will update the change  and if i use "="  i have to even add   IMAGE_CMD_TAR:append  let me raise the change .

Reply:
I think it is better to just use "=" instead here directly, as it won't change the previous behavior.